.changeset/small-buckets-cheat.md

```diff
@@ -0,0 +1,5 @@
+---
+"wrangler": patch
+---
+
+Support for Cloudchamber deployment labels
```

Comment:
this should follow the changesets format in our guidelines: https://github.com/cloudflare/workers-sdk/blob/main/CONTRIBUTING.md#changeset-message-format

Reply:
Thanks for catching it! Fixed.